java/src/main/java/be/tothepoint/Main.java

```diff
@@ -0,0 +1,42 @@
+package be.tothepoint;
```

Comment:
Can you rename the namespace? something with playing with projections? Or just dddeurope?